code/game/atoms.dm

```diff
@@ -567,9 +573,13 @@ its easier to just keep the beam vertical.
 		O.show_message(message,2,deaf_message,1)
 
 /atom/Entered(var/atom/movable/AM, var/atom/old_loc, var/special_event)
-	if(loc && MOVED_DROP == special_event)
-		AM.forceMove(loc, MOVED_DROP)
-		return CANCEL_MOVE_EVENT
+	if(loc)
+		for(var/i in AM.contents)
+			var/atom/movable/A = i
+			A.entered_with_container(old_loc)
+		if(MOVED_DROP == special_event)
```

Comment:
Please explain moved_drop, this may be duplicating functionality we already have

A while back i fixed up the dropped proc to only fire when an item actually leaves a mob, you can probably do anything you need to there

Reply:
i didnt add this it already was here

Reply:
you've used it though, why. What is this piece of code doing

if it was already there it may be deprecated